<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelect</class>
 <widget class="QWidget" name="ColorSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="redlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLabel" name="greenlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluelabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QSlider" name="redhorizontalSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QSlider" name="greenhorizontalSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="bluehorizontalSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="colorwidget" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>90</width>
     <height>100</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="redspinBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="greenspinBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="bluespinBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>redhorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redspinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenhorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenspinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bluehorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bluespinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
